.github/workflows/build_preconf.yml

```
@@ -2,34 +2,37 @@ name: Build
-        DOCKER_REG_PW: ${{ secrets.DOCKER_REG_PW }}
-        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-        REGISTRY_URL: ${{ secrets.REGISTRY_URL }}
-        REPO_NAME: ${{ secrets.REPO_NAME }}  
-      run: |
-        DOCKER_BUILDKIT=1 docker build -t helix:preconf --build-arg AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID --build-arg AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY --build-arg REPO_NAME=$REPO_NAME -f ./repos/$REPO_NAME/Dockerfile .
-        DOCKER_BUILDKIT=1 docker tag helix:preconf $REGISTRY_URL/helix:preconf
-        DOCKER_BUILDKIT=1 docker login https://$REGISTRY_URL/ -u "$DOCKER_REG_USR" -p "$DOCKER_REG_PW"
-        DOCKER_BUILDKIT=1 docker push $REGISTRY_URL/helix:preconf
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v2
```

actionlint

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)